Index.html

```diff
@@ -254,15 +261,19 @@ <h2>About Myself</h2>
               <hr>          
        </div>       
     </section>
-    <section class="contact">   
+    <section class="contact" id="contact">   
       <h1 >Contact me</h1>
       <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
       <form action="https://formspree.io/f/mqkoywbg" method="POST" class="pure-form" name="">            
-          <input type="text" class="input-rounded" placeholder="Lorem Ipsum" maxlength="30" name="name" required>                          
-          <input  type="email" class="input-rounded" placeholder="loremipsum@gmail.com" required>             
+          <label for='Name' class="visuallyhidden">Name</label>
+          <input type="text" class="input-rounded" placeholder="Name" maxlength="30" name="name" required>                          
+          <label for='email' class="visuallyhidden">Email Address</label>
+          <input  type="email" class="input-rounded" placeholder="Email Address" required>             
+          <label for='message'class="visuallyhidden">Message</label>
```

Comment:
- Hey :wave: , I see you've just added this <label> for Name, Email Address, and Message. You can add the labels but instead, the input should be nested in it. please, kindly remove the text content you've added to make your design consistent following the given Figma template. It should look like this. please, update this.
```
 <label for='email' class="visuallyhidden">
     <input  type="email" class="input-rounded" placeholder="Email Address" required>  
</label>
```

*Figma design*
![Screenshot from 2023-04-18 16-28-15](https://user-images.githubusercontent.com/59397159/232826539-4d5a9c4e-4c16-445d-90d5-63c63e474014.png)

*Your design*
![Screenshot from 2023-04-18 16-27-04](https://user-images.githubusercontent.com/59397159/232826542-c0d74490-f3f8-4adc-9ff8-69a67b74d657.png)